docs/appkit/features/smart-accounts.mdx

```diff
@@ -13,40 +13,78 @@ This update ensures a seamless and efficient experience for managing digital ass
+
+### What can I do with a Smart Account?
+
+Smart accounts unlock a host of use cases that were previously unavailable with EOAs. Essentially anything that can be programmed into a smart contract can be used by Smart Accounts.
+
+- **Automated Transactions:** Set up recurring payments or conditional transfers.
+- **Multi-Signature Authorization:** Require multiple approvals for a transaction to increase security.
+- **Delegated Transactions:** Allow a third party to execute transactions on your behalf under specific conditions.
+- **Enhanced Security:** Implement complex security mechanisms such as time-locked transactions and withdrawal limits.
+- **Interoperability:** Interact seamlessly with decentralized applications (dApps) and decentralized finance (DeFi) protocols.
+- **Custom Logic:** Create custom transaction rules and workflows that align with personal or business requirements.
+
+### How do I get a Smart Account?
+
+Existing AppKit Universal Wallet Users will be given the option to upgrade their account to a smart account. Once you upgrade you will still be able to access your EOA and self-custody your account. 
+
+New AppKit Universal Wallet Users will be given smart accounts by default when they login for the first time.
+
+### Does it cost anything?
+
+There is a small additional cost for activating your smart account. The activation fee is added to the first transaction and covers the network fees required for deploying the new smart contract onchain.
+
+### Can I export my Smart Account?
+
+No, you cannot export your Smart Account. The Smart Account (SA) is deployed by the EOA and is owned by the EOA. Your EOA account will always be exportable.
+Also is good to know that SA don't have seedphrases.
+
+### Can I rotate the signer ?
+
+Yes, you can rotate the signer manage it elsewhere if they want too. We expect there to be more UIs for this wallet in the future including Safe-Wallet
+
+### Can I withdraw all my funds from my Smart Account?
+
+Yes, you can withdraw all your funds from your Smart Account.
+
+### What are account names?
+
+Smart account addresses start with ’0x’ followed by 42 characters, this is the unique address of your smart account on the network. ‘0x’ addresses like this are long, unwieldy and unmemorable. AppKit allows you to assign a more memorable name for your smart account using [ENS](https://ens.domains/). 
+
+You can assign a name to your account and this will act as an alias for your account that can be shared publicly and provide a better user experience. AppKit account names are followed by the wcn.id. 
+
+### What can I do with my account name?
+
+As AppKit smart account addresses are the same across the supported networks, you only need one account name which can then be used across the networks. 
```

Comment:
I'd note that given we are using pimlico we are limited to their supported networks 
https://docs.pimlico.io/infra/platform/supported-chains